students/swietczak_monika/lesson_09_reading_and_writing_files/mad_libs.py

```diff
@@ -0,0 +1,17 @@
+words_to_find = ['ADJECTIVE', 'NOUN', 'VERB', 'NOUN2']
+
+filename = input("Enter a filename: ")
+text_file = open(filename, "r")
+content = text_file.read()
+text_file.close()
+
+for word in words_to_find:
+    content = content.replace(word, input("Enter an " + word.lower() + ': '),
+                              1)
```

Comment:
This formatting sucks, maybe store the input in a variable, as it gets too long.